backend/app/views/spree/admin/orders/cart.html.erb

```diff
@@ -2,6 +2,8 @@
 
 <%= render 'spree/admin/shared/order_tabs', current: :cart %>
 
+<h1><%= 'ABC' %></h1>
```

Comment:
@damianlegawiec Does this 'ABC' relates to something? I guess this is left by mistake or some other text needs to be here.